recipes/jupyter_nbextensions_configurator/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = 'jupyter_nbextensions_configurator' %}
+{% set version = '0.1.1' %}
+{% set md5 = 'c53ee4136529b915811cf0925154a291' %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/dc/4f/c08c6396ebd5cc8bf12a38892e52a7177abfd2e09c279ac2174d559fbc3f/{{ name }}-{{ version }}.tar.gz
```

Comment:
Sure. I tried that before, but it didn't work for me. I guess I just didn't wait long enough for `pypi.io` to pick up my changes from `pypi.python.org`. Will push an update now...